src/server/launchpad/resources.js

```diff
@@ -32,15 +42,15 @@ export class Resource {
 export class Root extends Resource {
   constructor(client, representation, uri) {
     super();
-    this.init(client, representation, uri);
+    this.init(client, uri, representation);
```

Comment:
It might be a good idea to switch the order of `uri` and `representation` in these inherited constructors' signatures at some point to match the new `Resource` constructor signature, to reduce the potential for confusion.